themes/bootstrap3/js/common.js

```diff
@@ -549,6 +549,131 @@ function bulkFormHandler(event, data) {
 }
 
 // Ready functions
+function setupStickyElements() {
```

Comment:
Rather than having a setupStickyElements function in the global namespace, it would probably be better to register this into the VuFind namespace as a module. I don't think much would need to change about the code, just a matter of changing the top-level to `VuFind.register('stickyElements', ...` and moving the initialization logic into an `init` method of the registered module. It should then bootstrap automatically as part of VuFind initialization without requiring a separate explicit call. See [check_item_statuses.js](https://github.com/vufind-org/vufind/blob/dev/themes/bootstrap3/js/check_item_statuses.js) for an example of this pattern. (This might also open the door to moving this into a sticky.js file that only gets included when sticky elements are turned on... though I'll leave that to your discretion).

Reply:
The js file is only included if any element is configured in the config.ini. We have to keep in mind that it would be possible to create elements with the sticky class without using the config. In that case the file would have to be included in the theme config.